linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_controller.js

```diff
@@ -5,63 +5,77 @@ var RouterController = (function () {
     yellows: {
       light: "#FFE7B3",
       tint: "#FAD78A",
-      yellow: "#ED9E64",
+      neutral: "#ED9E64",
       shade: "#D85B00",
       dark: "#B84D00"
     },
     greys : {
       light: "#F2F2F2",
       tint: '#C9C9C9',
-      grey: "#878787",
+      neutral: "#878787",
       shade: "#424242",
       dark: "#2B2B2B"
     },
     blues: {
       light: "#D1E2FB",
       tint: "#A4C4F1",
-      blue: "#709DDD",
+      neutral: "#709DDD",
       shade: "#4076C4",
       dark: "#163F79",
       night: "#0F2A50"
     },
     purples: {
       light: "#E1D1F6",
       tint: "#CAA2EA",
-      purple: "#9B4AD8",
+      neutral: "#9B4AD8",
       shade: "#6A18A4",
       dark: "#430880",
       night: "#2A084C"
     },
     greens: {
       light: "#D1F6E8",
       tint: "#A2EACF",
-      green: "#4AD8AC",
+      neutral: "#4AD8AC",
       shade: "#18A478",
       dark: "#08805B"
     },
     reds: {
       light: "#F6D1D1",
       tint: "#EAA2A2",
-      red: "#D84A4A",
+      neutral: "#D84A4A",
       shade: "#A41818",
       dark: "#800808"
     }
   }
 
-  //TODO: assign colorOrder colors to clients by client name
-  var colorOrder = [
-    colors.purples.purple,
-    colors.yellows.yellow,
-    colors.blues.blue,
-    colors.greens.green,
-    colors.reds.red,
-    colors.purples.shade,
-    colors.yellows.shade,
-    colors.blues.shade,
-    colors.greens.shade,
-    colors.reds.shade
+  var baseColorOrder = [
+    "purples",
+    "yellows",
+    "blues",
+    "greens",
+    "reds",
+    "purples.shade",
+    "yellows.shade",
+    "blues.shade",
+    "greens.shade",
+    "reds.shade"
   ];
 
+  var colorOrder = _.map(baseColorOrder, function(colorName) {
+    var color;
+    var colorParts = colorName.split(".");
+
+    if (colorParts.length > 1) {
+      color = colors[colorParts[0]][colorParts[1]];
+    } else {
+      color = colors[colorParts[0]].neutral;
+    }
```

Comment:
simpler as:
`var color = _.property(colorName)(color);`
would have to add .neutrals to the baseColorOrder